templates/guides/troubleshooting.md

```diff
@@ -28,10 +28,16 @@ The provider may return an error that looks similar to the following:
 │ Not Allowed, error={"detail":"Method Not Allowed"}
 ```
 
+or timeout when trying to create a resource.
```

Comment:
is this text supposed to go somewhere else?

Reply:
This was meant as an add-on to the error case above but it might just fit better into the related status codes description below. Mostly just wanted to include this context since this was the symptom I was seeing.

Reply:
ah, i got you now. no problem